blocks/api/post.pegjs

```diff
@@ -17,10 +17,25 @@ WP_Block_List
   = WP_Block*
 
 WP_Block
-  = WP_Block_Void
+  = WP_Tag_More
+  / WP_Block_Void
   / WP_Block_Balanced
   / WP_Block_Html
 
+WP_Tag_More
+  = "<!--" WS* "more" customText:(WS+ text:$((!(WS* "-->") .)+) { return text })? WS* "-->" noTeaser:(WS* WP_Tag_NoTeaser)?
```

Comment:
yeah I deferred to the Codex for the behavior. which should we fix? the Codex or Core?